src/SDKs/_metadata/cognitiveservices_data-plane_CustomImageSearch.txt

```diff
@@ -0,0 +1,11 @@
+2018-05-03 09:18:53 UTC
+
+1) azure-rest-api-specs repository information
+GitHub user: Azure
+Branch:      master
+Commit:      82502085032e822e2408d3e6f5d6a6e498f01c62
+
```

Comment:
Please check in the generate.cmd and generate.ps1 scripts

Reply:
Whats the generate.ps1? I am only aware of generate.cmd, similar to the one checked in for customsearch and other bing SDKs

Reply:
Added generate.cmd

Reply:
A generate.ps1 script should look something like [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/generate.ps1)
If there is a time crunch you can do this later.